ambari-web/app/views/wizard/step2_view.js

```diff
@@ -140,6 +131,32 @@ App.WizardStep2View = Em.View.extend({
      * @type {bool}
      */
     disabled: Em.computed.not('isEnabled')
-  })
+  }),
+
+  /**
+   * Filter for pre-registered hosts. Currently this is a text filter.
+   *
+   * Need to change it to column filter at some point
+   */
+  filterHostsTable: function () {
+    var self = this;
+
+    var filter = this.get('controller.filterText');
+    if (filter) {
+      filter = filter.toUpperCase();
+    }
+    this.get('controller.manuallyInstalledHosts').forEach(function (host) {
+      var element = self.$('tr[id="' + host.hostName + '"]')
+      if (element) {
```

Comment:
@atkach I am changing the display of each item in the list to none or block based on what is being entered in the filter box. DOM being fetched is using its id. Do you think there is a way to use class bindings in this scenario. Please suggest.

Reply:
For example, define host as an object with the flag "isFiltered", in the template:
 {{#each host in view.hosts}}
              <tr {{bindAttr class="host.isFiltered::hidden"}}>